docs/src/decision-programming/cvar.md

```diff
@@ -0,0 +1,118 @@
+# [Conditional value-at-risk](@id cvar)
+
+## RJT model
+
+CVaR formulation for the RJT model is close to that of path-based model. A diagram can have only a single value node, when using RJT-based CVaR. Trying to call the RJT-based CVaR function using a diagram with more than one value node results in an error.
+
+We denote the possible utility values with $u ∈ U$ and suppose we can define the probability $p(u)$ of attaining a given utility value. In the presence of a single value node, we define $p(u) = ∑_{s_{C_v}∈ \text{\{} S_{C_v} \vert U(s_{C_v})=u \text{\}} }µ(s_{C_v})$. We can then pose the constraints
+
+$$η-u≤M λ(u),\quad ∀u∈U \tag{26}$$
+
+$$η-u≥(M+ϵ) λ(u) - M,\quad ∀u∈U \tag{27}$$
+
+$$η-u≤(M+ϵ) \bar{λ}(u) - ϵ,\quad ∀u∈U \tag{28}$$
+
+$$η-u≥M (\bar{λ}(u) - 1),\quad ∀u∈U \tag{29}$$
+
+$$\bar{ρ}(u) ≤ \bar{λ}(u),\quad ∀u∈U \tag{30}$$
+
+$$p(u) - (1 - λ(u)) ≤ ρ(u) ≤ λ(u),\quad ∀u∈U \tag{31}$$
+
+$$ρ(u) ≤ \bar{ρ}(u) ≤ p(u),\quad ∀u∈U \tag{32}$$
+
+$$∑_{u∈U}\bar{ρ}(u) = α \tag{33}$$
+
+$$\bar{λ}(u), λ(u)∈\{0, 1\},\quad ∀u∈U \tag{34}$$
+
+$$\bar{ρ}(u),ρ(u)∈[0, 1],\quad ∀u∈U \tag{35}$$
+
+$$η∈\mathbb{R} \tag{36}$$
+
+where where α is the probability level in CVaR<sub>α</sub>.
+
+$CVaR_α$ can be obtained as $1/α ∑_{u∈U} \bar{ρ}(u)u$.
+
+More details, including explanations of variables and constraints, can be found from Herrala et al. (2024)[^4].
+
+## Path-based model
```

Comment:
I think we could present the path based model CVaR first since this section contains some information that is useful for understanding the RJT-formulation as well.

Reply:
Order changed and some clarification